smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/CommandGenerator.java

```diff
@@ -205,7 +200,7 @@ private void writeSerde() {
                 .indent()
                     .write("input: $L,", inputType)
                     .write("protocol: string,")
-                    .write("utils?: Utils")
+                    .write("context?: SerdeContext")
```

Comment:
```suggestion
                    .write("context: SerdeContext")
```
This should not be optional.